content/id/docs/tutorials/kubernetes-basics/_index.html

```diff
@@ -79,13 +81,33 @@ <h2>Kubernetes Basics Modules</h2>
           <div class="row">
             <div class="col-md-4">
               <div class="thumbnail">
-                <a href="/docs/tutorials/kubernetes-basics/expose/expose-intro/"><img src="/docs/tutorials/kubernetes-basics/public/images/module_04.svg?v=1469803628347" alt=""></a>
+                <a href="/id/docs/tutorials/kubernetes-basics/expose/expose-intro/"><img src="/docs/tutorials/kubernetes-basics/public/images/module_04.svg?v=1469803628347" alt=""></a>
+                <div class="caption">
+                  <a href="/id/docs/tutorials/kubernetes-basics/expose/expose-intro/"><h5>4. Mengekspos aplikasimu secara publik</h5></a>
+                </div>
+              </div>
+            </div>
+            <div class="col-md-4">
+              <div class="thumbnail">
+                <a href="/docs/tutorials/kubernetes-basics/scale/scale-intro/"><img src="/docs/tutorials/kubernetes-basics/public/images/module_05.svg?v=1469803628347" alt=""></a>
+                <div class="caption">
+                  <a href="/docs/tutorials/kubernetes-basics/scale/scale-intro/"><h5>5. Scale up your app</h5></a>
+                </div>
+              </div>
+            </div>
+            <div class="col-md-4">
+              <div class="thumbnail">
+                <a href="/docs/tutorials/kubernetes-basics/update/update-intro/"><img src="/docs/tutorials/kubernetes-basics/public/images/module_06.svg?v=1469803628347" alt=""></a>
+                <div class="caption">
+                  <a href="/docs/tutorials/kubernetes-basics/update/update-intro/"><h5>6. Update your app</h5></a>
```

Comment:
Missed translation?
```suggestion
                  <a href="/docs/tutorials/kubernetes-basics/update/update-intro/"><h5>6. Memperbarui aplikasi kamu</h5></a>
```

Reply:
Since the linked article is still in English, I was not sure if we need to translate it.
```suggestion
                  <a href="/docs/tutorials/kubernetes-basics/update/update-intro/"><h5>6. Memperbarui aplikasimu</h5></a>
```

Reply:
We can translate and update the links once our indonesian pages are ready